util/test.js

```diff
@@ -1,3 +1,10 @@
+const getCode = async (bundle) => {
```

Comment:
`getCode` should be used from `testBundle`

Reply:
Not necessarily. Leaving it separate to evaluate usefulness.